interface/billing/sl_eob_search.php

```diff
@@ -1104,8 +1113,7 @@ class="fa fa-info-circle oe-superscript"
                                            onclick="return npopup(<?php echo attr_js($row['pid']); ?>)"><?php echo text($row['lname']) . ', ' . text($row['fname']); ?></a>
                                     </td>
                                     <td class="detail">&nbsp;
-                                        <a href="sl_eob_invoice.php?isPosting=1&id=<?php echo attr_url($row['id']); ?>"
-                                           target="_blank"><?php echo text($row['pid']) . '.' . text($row['encounter']); ?></a>
+                                        <a onclick="editInvoice(event,<?php echo attr_url($row['id']); ?>)"><?php echo text($row['pid']) . '.' . text($row['encounter']); ?></a>
```

Comment:
ok, this will seem crazy :)
When doing a js param within html attributed, then do:
```
<a onclick="editInvoice(event,<?php echo attr_js($row['id']); ?>)"><?php echo text($row['pid']) . '.' . text($row['encounter']); ?></a>
```